addons/mail/static/tests/emoji/emoji.test.js

```diff
@@ -39,6 +39,10 @@ test("search emoji from keywords", async () => {
     await click("button[aria-label='Emojis']");
     await insertText("input[placeholder='Search emoji']", "mexican");
     await contains(".o-Emoji", { text: "🌮" });
+    await insertText("input[placeholder='Search emoji']", "9", { replace: true });
+    await contains(".o-Emoji:eq(0)", { text: "🕘" });
+    await contains(".o-Emoji:eq(1)", { text: "🕤" });
+    await contains(".o-Emoji:eq(2)", { text: "9️⃣" });
```

Comment:
"Improved" fuzzy search put priority in clocks, due to them being earlier in emoji list and they have exactly "9" in keywords, whereas 9️⃣ doesn't even have "9" and just `keycap: 9` in keywords.

Improving emoji data / fuzzy-search weight is out-of-scope of PR. Besides the precedence of clocks matches the behaviour of competitors...